core/templates/pages/blog-dashboard-page/modal-templates/upload-blog-post-thumbnail.component.ts

```diff
@@ -94,7 +97,7 @@ export class UploadBlogPostThumbnailComponent implements OnInit {
   }
 
   reset(): void {
-    this.uploadedImage = null;
+    this.uploadedImage = false;
```

Comment:
Did you check that this does not break the behaviour of the uploader?

Reply:
yes checked